sys/slot_util/Makefile

```diff
@@ -0,0 +1,3 @@
+CFLAGS += -DRIOTBOOT_LEN=$(RIOTBOOT_LEN)
```

Comment:
This variable should not be `RIOTBOOT` as it is  for`SLOT_UTIL`. See also the global comment.